docs/how-to-guides/renkulab/ssh-into-sessions.rst

```diff
@@ -0,0 +1,256 @@
+---------------------------------------
+
+You can check if your project supports SSH by clicking the drop down next to the
+Session Start button and seeing if the SSH option is visible and enabled. If you
+see the SSH-enabled icon in the screenshot below, your project is SSH-enabled,
+and you can skip down to the next section!
+
+.. warning::
+    <screenshot of the Session Start drop down menu with SSH enabled>
+
+.. note::
+    **Don't see an SSH option in the Session Start menu?** If you are working on
+    a custom instance of RenkuLab (not renkulab.io), this may be because SSH
+    needs to be enabled by your administrator. Contact your RenkuLab
+    administrator to see if SSH can be enabled.
+
+Enabling SSH for your project
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. warning::
+    <screenshot of the Session Start drop down menu with SSH NOT enabled>
+
+If the SSH dialogue says that SSH is not enabled for this project, this likely
+means that your project is using an old project template. To update your project
+template, go to the project "Overview" tab and open the "Status" section. Find
+the "Template Version" section, and click "Update".
+
+.. warning::
+
+    <screenshot of the Status page with a Template Version box that's asking to be updated>
+
+Alternatively, you can manually enable SSH for your project by :ref:`upgrading your
+base image <renku_base_image_upgrade>` to at least version 0.14.0. Then, run
+``renku config set ssh_supported true`` to let Renku know that your project supports
+SSH connections.
+
+.. note::
+
+    **Are you using a custom project template?** If you've updated your project
+    template but your project still doesn't support SSH, ask the template
+    maintainer to enable SSH in the template and perform the update again.
+
+.. note::
+
+    **Note for template maintainers:** See the documentation on the Template
+    :ref:`manifest-yaml` for details on how to enable SSH support in your
+    template, specifically the details on the ``ssh_supported`` flag.
+
+
+Set up your local system for SSH access
+---------------------------------------
+
+Connecting to RenkuLab sessions via SSH requires 3 prerequisites to be setup on
+your local machine (where you want to SSH from):
+
+* OpenSSH is installed
+* the Renku CLI is installed, with version >= <X>
+* your Renku project is cloned
+
+This section will guide you through setting up these prerequisites.
+
+    .. warning::
+        <Enter version number above 1x>
+
+#.  To use the SSH feature, you need to have OpenSSH installed on your system.
+
+    .. warning::
+        <How to check is OpenSSH is installed?!>
+
+#.  Install the Renku CLI version >= <X>.
+
+    If you do not already have the CLI installed, see :ref:`cli_installation`. 
+   
+    To check the version of your Renku CLI, run ``renku --version``. If the
+    version is less than <X>, run ``pip install --upgrade renku>=<X>``.
+
+    .. warning::
+        <Enter version number above 3x>
```

Comment:
@Panaetius TODO: Fill in Renku CLI version numbers once known

Reply:
@Panaetius I filled in the version numbers with `renku>=2.3.0` - is that correct?